docs/index.md

```diff
@@ -6,4 +6,6 @@ sidebar_label: Welcome
 
 Welcome to the open.mp wiki!
 
+This is a work-in-progress site aimed at providing an easily accessible, easy to contribute to documentation source for open.mp and SA-MP. Some SA-MP specific pages are not here yet and can be found [on the old wiki](https://wiki.sa-mp.com/). If you believe a page from the old wiki should be updated, please open an issue to let us know or even better, a pull-request with the newly updated page!
```

Comment:
> If you believe a page from the old wiki should be updated

Maybe a better word would be 'added' rather than 'updated', as you can't update a page that doesn't exist yet